### PR TITLE
Fix typos, grammar, and improve readme formatting

### PR DESCRIPTION
**Pull request by Alhadis**
This PR does exactly as advertised. =) And by "PR", I mean `pedantic revisions`.

ECMAScript is officially typeset with five leading majuscules, which was the first thing I fixed. Then I kinda got carried away polishing other typos and improving overall grammar.

~~As I joke, I also reformatted your code to use tabs instead of spaces~~... almost did, hahaha.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -3,13 +3,16 @@
...
 echo "[variable1, variable2, variable3];" | eswrap --limit 20
-[
+
+# Output:
+'[
   variable1,
   variable2,
   variable3,
-];
+];'
```
Comment (kevinastone) [inline Readme.md]: why the extra single quote?

**Comment by kevinastone** [comment]
Thanks for the fixes.  This is an experimental personal project that isn't really ready for use (yet).

File: Readme.md (hunk 1 above)
Comment (Alhadis) [inline Readme.md]: Oh! That's the closing quote for the string that begins under the `# Output` line... I enclosed it in a string to make the highlighted syntax look uniform (as opposed to shell-script syntax). You can see how [it looks here](https://github.com/Cutlery-Drawer/eswrap/blob/f2cf0128bb5710158ffb1c0f2210e7c2eb05e04e/Readme.md). =)

Comment (kevinastone) [inline Readme.md]: Ok, but the command doesn't actually quote the output:
```
kstone@kstone-mbp ~> echo "[variable1, variable2, variable3];" | eswrap --limit 20
[
  variable1,
  variable2,
  variable3,
];⏎ 
```

**Comment by Alhadis** [comment]
Haha, I could tell. ^^ But I was waiting for a Vagrant box to download/install (and I felt slightly bad after replying to your CSS issue over at Atom, so...).

**Comment by Alhadis** [comment]
Also, I'm compelled by an uncontrollable force to correct incorrect capitalisation of ECMAScript and JavaScript.

My high school English teacher loved it when I corrected his typos. Honest.

File: Readme.md (hunk 1 above)
Comment (Alhadis) [inline Readme.md]: Yeah, on second thought, it was pretty dumb of me to use that as a hack to control highlighting colour (and hell, the braces and semicolon aren't [coloured on GitHub anyway](https://github.com/Cutlery-Drawer/eswrap/blob/fix-typos/Readme.md)). Sorry.

Fixed!

**Review by kevinastone — APPROVED** [review]

**Pull request merged**